ga4gh/client.py

```diff
@@ -99,8 +103,14 @@ def _deserializeResponse(self, response, protocolResponseClass):
         jsonResponseString = response.text
         self._updateBytesRead(jsonResponseString)
         self._debugResponse(jsonResponseString)
-        responseObject = protocolResponseClass.fromJsonString(
-            jsonResponseString)
+        try:
+            responseObject = protocolResponseClass.fromJsonString(
+                jsonResponseString)
+        except protocol.UnableToDecodeJsonException as exception:
```

Comment:
This seems even more complicated! Is this necessary for the getVariant functionality? If not, I think we should put it into a separate PR.